serializers/token.go

```diff
@@ -0,0 +1,9 @@
+package serializers
+
+type TokenResponse struct {
+	Id           int64    `jsonapi:"primary,token"`
+	AccessToken  string `jsonapi:"attr,access_token"`
+	ExpiresIn    int64  `jsonapi:"attr,expires_in"`
```

Comment:
@junan Should we keep the original type for this to `time.duration` even that it's `int64` in the `time/time.go`?
Because i think it looks more meaningful.
You can keep it if you need like this 😛